modules/TestUtils.js

```diff
@@ -8,6 +8,32 @@ export const isFunction = (object) =>
   typeof object === 'function'
 
 /**
+ * Returns true if the given object is an array.
+ */
+export const isArray = (object) =>
+  Array.isArray(object)
+
+/**
+ * Returns true if the given object is an object.
+ */
+export const isObject = (object) =>
+  Object.prototype.toString.call(object) === '[object Object]'
+
+/**
+ * Returns true if the given object is an instanceof value
+ * or its typeof is the given value.
+ */
+export const isA = (object, value) => {
+  if (isFunction(value))
+    return object instanceof value
+
+  if (value === 'array')
+    return Array.isArray(object)
```

Comment:
I actually didn't change that function, just moved it up. ESLint needed no function use before definition, and since I used `isObject` in `objectContains` I moved all the `is*` up.